src/Gtk/Avalonia.Gtk3/KeyTransform.cs

```diff
@@ -182,17 +187,18 @@ static class KeyTransform
             //{ GdkKey.?, Key.LaunchApplication2 }
             { GdkKey.semicolon, Key.OemSemicolon },
             { GdkKey.plus, Key.OemPlus },
+            { GdkKey.equal, Key.OemPlus },
             { GdkKey.comma, Key.OemComma },
             { GdkKey.minus, Key.OemMinus },
             { GdkKey.period, Key.OemPeriod },
-            { GdkKey.slash, Key.Oem2 }
-            //{ GdkKey.?, Key.OemTilde }
+            { GdkKey.slash, Key.Oem2 },
+            { GdkKey.grave, Key.OemTilde },
             //{ GdkKey.?, Key.AbntC1 }
             //{ GdkKey.?, Key.AbntC2 }
-            //{ GdkKey.?, Key.Oem4 }
-            //{ GdkKey.?, Key.OemPipe }
-            //{ GdkKey.?, Key.OemCloseBrackets }
-            //{ GdkKey.?, Key.Oem7 }
+            { GdkKey.bracketleft, Key.OemOpenBrackets },
+            { GdkKey.backslash, Key.OemPipe },
```

Comment:
Pipe and backslash are on the same key with pipe being the 'uppercase' marking on the key.
It is interesting that sometimes the keys are named after the uppercase variant, sometimes after the lower.   Things get even more confusing because the gdk enum might use the lowercase name and the key ednum the upper.
Another example of this is where GdkKey.grave maps to Key.OemTilde -- once again the gdk enum is the lowercase variant and the Key enum is the upper.
This is why I've only been mapping keys that I can test in both windows and linux to be sure I'm not getting things mixed up.

Reply:
Ahh now that i look on it, those two are on the same key indeed. I thought they have separate keycodes hahaha.. this stuff is quite confusing indeed